deepspeed/pt/deepspeed_cuda.py

```diff
@@ -403,11 +403,14 @@ class DeepSpeedTransformerLayer(nn.Module):
 
             initial_biases: Optional: Only used for unit test
     """
-    def __init__(self, layer_id, config, initial_weights=None, initial_biases=None):
+    layer_id = 0
+
```

Comment:
Docstring should be updated accordingly.

Reply:
sure, I will do that. thanks